src/Numerics/DGMethods/DGModel_kernels.jl

```diff
@@ -1518,6 +1518,9 @@ end
         @unroll for s in 1:num_state_prognostic
             state[n, s, e] = l_state[s]
         end
+        @unroll for s in 1:num_state_auxiliary
+            state_auxiliary[n, s, e] = local_state_auxiliary[s]
+        end
```

Comment:
I'm not sure that this is a good idea. Looping in @kpamnany.

Reply:
This is clearly telling us that we need to go ahead with that split of `state_auxiliary` into a prognostic state independent auxiliary state and a prognostic state dependent auxiliary state. The very first thing to do is come up with suitable names for these two auxiliary states.

I think it is fine for `init_state_prognostic` itself to initialize the prognostic state dependent auxiliary state (and it is probably more efficient as well). If we follow this, then we can use `init_state_auxiliary` for the initialization of the prognostic state independent auxiliary state and don't have to add any new interface calls -- just rename them appropriately.

Reply:
`AuxStatic` and `AuxDynamic`?

Reply:
They aren't really "static" or "dynamic" though? @simonbyrne, opinion?

Reply:
I think the ones that don't depend on the prognostic state are only computed once, whereas ones that do depend on the prognostic state are continually updated.

Reply:
This split is best left to another PR. I think this change is okay in the meantime.